INSTALL.md

```diff
@@ -176,3 +184,5 @@ Start up the rails server by running the following command:
 $ UNICORN_WORKERS=2 rails server -b 0.0.0.0 # Runs the server, available at localhost:3000
 ```
 > The number of unicorn workers is variable and sometimes it will need more than 2. In case the server is slow or start suffering from timeouts, try restarting porta with a higher number like 8.
+
+**NOTE:** 3scale porta will resolve all subdomains locally through use of `.lvh.me` domain extension.
```

Comment:
Probably add an example.

When you seed the application with `rake db:setup` it will create the provider account and the master account, respectively accessible through, `http://provider-admin.example.com.lvh.me:3000` and `http://master-account.example.com.lvh.me:3000`

Reply:
Maybe this note would be better right after the `db:setup` command, and mentioning also that password should wrote down at that moment.

Reply:
Yeah that's a good point @josemigallas as I did have to reset password having missed that output on the terminal. I can also add an example of the domains which are seeded @hallelujah